README.md

```diff
@@ -3,31 +3,65 @@ Profile
 
 Supports configurable user profiles.
 
+This module is the successor to the [Drupal 7 Profile2 module](https://drupal.org/project/profile2).
+
 Installation
 -------------
 
- * Copy the whole profile directory to your modules directory and
-   activate the module.
-
+* Install this module using the official Backdrop CMS instructions at
+  https://backdropcms.org/guide/modules.
 
 Usage
 -----
-   
- * Go to /admin/structure/profiles for managing profile types.
- * By default users may view their profile at /user and edit them at
-   'user/X/edit'.
 
-LICENSE
----------------    
+* Visit the configuration page under Administration > Structure >
+  Profiles (admin/structure/profiles) for managing profile types.
+
+* By default users may view their profile at /user and edit them at
+   user/N/edit.
+
+Upgrading from Drupal 7
+-----------------------
+
+This module is the successor to the [Drupal 7 Profile2
+module](https://drupal.org/project/profile2) and will automatically import and
+convert any Profile2 profiles when upgrading from a Drupal 7 installation.
+
+Note that this module does NOT import from the [deprecated Profile
+module](https://www.drupal.org/node/874026) that was provided in Drupal 7 core
+as an upgrade path from Drupal 6 sites that used Profiles. If you have profiles
+from the Drupal 7 Profile module, you should either convert them to Profile2
+profiles in Drupal 7 prior to upgrading to Backdrop, or recreate them in
+Backdrop after you upgrade.
+
+Documentation
+-------------
+
+Additional documentation is located in the [Wiki](https://github.com/backdrop-contrib/me/wiki/Documentation).
```

Comment:
Shouldn't this URL be pointing to "profile" and not "me"?

Reply:
Yes, fixed.